ApollosProject/apollos-apps
# Update PrayerExperienceSwiper.js

## ✏️ Description

This PR adds the ability to "pray" for a prayer request within the Daily Habits Prayer Experience.

## 🔬 To Test
1. Connect you project to the production dB
2. `yarn && yarn start` and `yarn iOS`
3. Navigate to the Prayer Experience in the Daily tab
4. Pray for a prayer
5. Check the dB for a record in the Interactions table

## 📸 Screenshots

https://user-images.githubusercontent.com/45076058/203460563-83489ddc-efae-45bc-8f51-11f856713aa9.mp4

## 📝 Checklist
